cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -160,8 +159,8 @@ Recommendations:
   used by Cluster Autoscaler, depending on whether your ASG utilizes Launch
   Configurations or Launch Templates.
 * If Cluster Autoscaler adds a node to the cluster, and the node has taints applied
-  when it joins the cluster that Cluster Autoscaler was unaware of (because the tag
-  wasn't supplied), this can lead to significant confusion and misbehavior.
+  when it joins the cluster that Cluster Autoscaler was unaware of, because the tag
+  wasn't applied to the ASG and/or supplied to the `--node-group-auto-discovery` flag, it can lead to significant confusion and misbehavior. For example, if Cluster Autoscaler adds a node to help schedule a pending pod, and the node turns up with a taint that prevents the pod from being scheduled onto it, that can cause issues with pod scheduling.
```

Comment:
```suggestion
  wasn't applied to the ASG and/or supplied to the `--node-group-auto-discovery` flag, it can lead to significant confusion and misbehavior. For example, if Cluster Autoscaler adds a node to help schedule a pending pod, and the node turns up with a taint that prevents the pod from being scheduled onto it, this can prevent pods being scheduled.
```
?